crates/sui-graphql-rpc/schema/current_progress_schema.graphql

```diff
@@ -517,24 +532,81 @@ type NameServiceEdge {
 }
 
 type Object implements ObjectOwner {
+	"""
+	If this object is a MoveObject, this is a number that increases each time a tx takes this object as a mutable input
+	This is a lamport timestamp, not a sequentially increasing version
+	If this object is a MovePackage, the version is increased every time the package is upgraded
+	Note that packages are referred to by move calls using just their ID, and they are always loaded at their latest version
+	"""
 	version: Int!
+	"""
+	The object digest is a 32-byte hash of the sui_type::Object struct
+	This uniquely identifies the object
```

Comment:
- We shouldn't refer to internal types like `sui_types::Object` in externally facing documentation
- The digest identifies the object's contents, not the object itself (the ObjectID uniquely identifies the object)

```suggestion
	32-byte hash that identifies the object's current contents, encoded as a Base58 string.
```